src/types.js

```diff
@@ -0,0 +1,26 @@
+const negativeInf = Number.NEGATIVE_INFINITY
+const positiveInf = Number.POSITIVE_INFINITY
```

Comment:
Out of curiosity, why did you use these constants instead of `-Infinity`/`Infinity`?

Reply:
They just feel more convenient, so for me the rule of thumb is to use constant (`Number.POSITIVE_INFINITY`) instead of real value (`Infinity`).